content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -5378,13 +5386,21 @@ Example parameter values:
 
 See [middleware examples](/orm/prisma-client/client-extensions/middleware#samples) .
 
-### `$executeRaw()`
+### `$queryRaw`
```

Comment:
Does this not have `()` because it's a template string?

Reply:
Yes, exactly. There is no great way to represent that that I could find :/